src/utils.py

```diff
@@ -0,0 +1,17 @@
+def parse_msg_json(msg):
+  """This parses the json string from the message field.
+  """
```

Comment:
why are the docs ending quotes on the next line. is this python standard? it looks kinda wacky